presto-main/src/main/java/com/facebook/presto/util/JsonUtil.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.util;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.base.Throwables;
+
+import java.io.IOException;
+import java.io.StringWriter;
+import java.io.Writer;
+
+import static java.util.Objects.requireNonNull;
+
+public class JsonUtil
```

Comment:
I don't think we need a generic "json utils" just for this. Since it's used in a single place, move it there. We can always generalize later if necessary.